core/src/main/java/org/frankframework/pipes/IsUserInRolePipe.java

```diff
@@ -58,39 +61,49 @@ public void configure() throws ConfigurationException {
 		if (StringUtils.isNotEmpty(getNotInRoleForwardName())) {
 			notInRoleForward = findForward(getNotInRoleForwardName());
 			if (notInRoleForward==null) {
-				throw new ConfigurationException("notInRoleForwardName ["+getNotInRoleForwardName()+"] not found");
+				throw new ConfigurationException("notInRoleForwardName not found");
```

Comment:
@tnleeuw @jkosternl dit is in de review niet opgemerkt maar bij nader inzien is dit niet toch niet zo handig.

Ik ging er vanuit dat als de `notInRoleForward` null is, het geen zin heeft om dan nog te vermelden om welke naam het ging.

Echter bij een tweede blik hierop blijkt dat het toch voor kan komen dat `getNotInRoleForwardName()` een gevulde string is en dat de gerelateerde forward niet gevonden kan worden.

Het lijkt mij een goed idee om dit te herstellen naar `throw new ConfigurationException("could not find forward for notInRoleForwardName ["+ getNotInRoleForwardName()+"]");`

Reply:
@Laurens-makel Maak jij daarvoor dan een 2e PR aan?

Reply:
#6105